.mise.toml

```diff
@@ -132,8 +132,13 @@ ruff format --check
 find . -type f -name "Tiltfile" | while read -r file; do
   ruff format --check --diff "$file"
 done
+mypy . --ignore-missing-imports --check-untyped-defs --install-types --non-interactive
 """
 
+[tasks."mypy"]
+description = "Run mypy"
+run = "mypy . --ignore-missing-imports --check-untyped-defs --install-types --non-interactive"
+
 [tasks."bootstrap:kubeconfig"]
```

Comment:
I think this should be renamed to `[tasks.lint]`
Like we've got `[tasks.fmt]` and `[tasks."fmt:check"]`